src/extensions/toolbar.ts

```diff
@@ -81,7 +81,7 @@ export class Toolbar {
                         // hash that was in the url before the redirect
                         location.hash = state['desiredHash']
                     } else if (history) {
-                        history.replaceState('', document.title, location.pathname + location.search) // completely remove hash
+                        history.replaceState(history.state, document.title, location.pathname + location.search) // completely remove hash
```

Comment:
```suggestion
                        // second param is unused see https://developer.mozilla.org/en-US/docs/Web/API/History/replaceState
                        history.replaceState(history.state, "", location.pathname + location.search) // completely remove hash
```

seems reasonable based on https://developer.mozilla.org/en-US/docs/Web/API/History/replaceState and a little bit of fun that the second parameter is unused but exists for historical reasons 🤣